src/T2DMap.cpp

```diff
@@ -2969,11 +2969,35 @@ bool T2DMap::event(QEvent* event)
     return QWidget::event(event);
 }
 
+bool T2DMap::isClickInEmptySpace(const QPoint& pos) {
+    const float mx = (pos.x() / mRoomWidth) + mMapCenterX - (xspan / 2.0);
+    const float my = (yspan / 2.0) - (pos.y() / mRoomHeight) - mMapCenterY;
+
+    for (const auto& roomId : mpMap->mpRoomDB->getArea(mAreaID)->getAreaRooms()) {
+        TRoom* room = mpMap->mpRoomDB->getRoom(roomId);
+        if (!room) continue;
+
+        if (fabs(mx - room->x()) < mRoomWidth / 2.0 && fabs(my - room->y()) < mRoomHeight / 2.0) {
+            return false;
+        }
+    }
+    return true;
+}
+
+
```

Comment:
Code style fixes to confirm to "our" style:
```suggestion
bool T2DMap::isClickInEmptySpace(const QPoint& pos)
{
    const float mx = (pos.x() / mRoomWidth) + mMapCenterX - (xspan / 2.0);
    const float my = (yspan / 2.0) - (pos.y() / mRoomHeight) - mMapCenterY;

    for (const auto& roomId : mpMap->mpRoomDB->getArea(mAreaID)->getAreaRooms()) {
        TRoom* room = mpMap->mpRoomDB->getRoom(roomId);
        if (!room) {
            continue;
        }

        if (fabs(mx - room->x()) < mRoomWidth / 2.0 && fabs(my - room->y()) < mRoomHeight / 2.0) {
            return false;
        }
    }
    return true;
}


```